src/client.coffee

```diff
@@ -72,13 +72,19 @@ class SlackClient
   ###
   send: (envelope, message) ->
     message = @format.outgoing(message)
+    room = envelope.room
+    if !(room.match /C0[A-Z0-9]{7}/)
```

Comment:
Channels begin with `C` not `C0`

see https://github.com/slackhq/node-slack-sdk/blob/master/lib/models/helpers.js#L28